spec/models/service_template_transformation_plan_spec.rb

```diff
@@ -37,6 +37,18 @@
     }
   end
 
+  describe '.public_service_templates' do
+    it 'display public service templates' do
+      st1 = FactoryGirl.create(:service_template_transformation_plan)
+      st2 = FactoryGirl.create(:service_template)
+
+      expect(st1.internal).to be_truthy
```

Comment:
Prefer `internal?` since it's a boolean attribute

Reply:
On second thought... Do you need to test this?  It's just testing `default_value_for`, right?

Reply:
Yes, they are testing the default value, to make sure if it is set correctly.  `internal` is column name.